### ISSUE TITLE
Unable to build

### ISSUE THREAD

**Issue by yurrriq** [lfex/clj#25]
rvirding/lfe@d9a4e4d26a40ddae5aea09244fcd7260cac3f4ac
lfex/clj@b935818364ff9b2715345a050c97db81289ae628
lfe-rebar3/compile@6d7a8245f0979c0a3d5b6cd3b0f148ed16b1039c

```
rebar 3.0.0+build.3378.refc619921 on Erlang/OTP 18 Erts 7.3
```

``` erlang
===> Uncaught error: {case_clause,
                             {error,
                              [{error,
                                [{"/path/to/my_project/_build/default/lib/clj/src/clj-seq.lfe",
                                  [{3,lfe_lint,
                                    {redef_fun,
                                     {'LFE-EXPAND-EXPORTED-MACRO',3}}}]}],
                                []}],
                              [],[]}}
```

**Comment by yurrriq** [comment]
N.B. This is likely due to a stale `./apps/*/rebar.lock`.

**Comment by yurrriq** [comment]
All the `<<"clj">>` entries match...

**Comment by rvirding** [comment]
Is LFE-EXPAND-EXPORTED-MACRO/3 being explicitly defined anywhere?

Robert

All the <<"clj">> entries match...

**Comment by yurrriq** [comment]
I'm pretty sure this is fixed already (see discussion in #22). I must've had a stale [kla](https://github.com/lfex/kla) somewhere on the code path.
